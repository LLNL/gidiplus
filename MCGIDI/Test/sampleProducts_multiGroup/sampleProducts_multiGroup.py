# <<BEGIN-copyright>>
# Copyright 2019, Lawrence Livermore National Security, LLC.
# See the top-level COPYRIGHT file for details.
# 
# SPDX-License-Identifier: MIT
# <<END-copyright>>

from __future__ import print_function

import sys
import os

if( not( os.path.exists( 'Outputs' ) ) ) : os.mkdir( 'Outputs' )

output = ''
cmd = ''
sep = './'
for arg in sys.argv[1:] :
    cmd += sep + arg
    sep = ' '
    if( os.sep in arg ) : continue
    output += '.' + arg
output = output[1:]

benchmark = os.path.join( 'Benchmarks', output + '.out' ).replace( '..', '.' )
output = os.path.join( 'Outputs', output + '.out' ).replace( '..', '.' )

if( os.system( cmd + ' > ' + output ) != 0 ) : raise Exception( 'FAILURE: %s' % cmd )
cmd = '../Utilities/diff.com sampleProducts_multiGroup/%s %s %s' % ( sys.argv[1], benchmark, output )
os.system( cmd )
