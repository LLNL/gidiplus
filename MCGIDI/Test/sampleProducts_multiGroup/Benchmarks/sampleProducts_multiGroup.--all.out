label = MultiGroup_000  temperature = 2.586e-08
label = MultiGroup_001  temperature = 1e-06
label = MultiGroup_002  temperature = 0.0001
reaction (  0) = n + O16  threshold = 0
    energy = 1e-13
        productIndex            n KE = 8.11435e-14
        productIndex          O16 KE = 1.88565e-14
    energy = 2e-13
        productIndex            n KE = 1.94387e-13
        productIndex          O16 KE = 5.61302e-15
    energy = 4e-13
        productIndex            n KE = 3.33666e-13
        productIndex          O16 KE = 6.63338e-14
    energy = 8e-13
        productIndex            n KE = 7.7834e-13
        productIndex          O16 KE = 2.16602e-14
    energy = 1.6e-12
        productIndex            n KE = 1.40046e-12
        productIndex          O16 KE = 1.99543e-13
    energy = 3.2e-12
        productIndex            n KE = 2.86795e-12
        productIndex          O16 KE = 3.32047e-13
    energy = 6.4e-12
        productIndex            n KE = 6.31761e-12
        productIndex          O16 KE = 8.23912e-14
    energy = 1.28e-11
        productIndex            n KE = 1.03551e-11
        productIndex          O16 KE = 2.44491e-12
    energy = 2.56e-11
        productIndex            n KE = 2.26193e-11
        productIndex          O16 KE = 2.98071e-12
    energy = 5.12e-11
        productIndex            n KE = 4.20967e-11
        productIndex          O16 KE = 9.10329e-12
    energy = 1.024e-10
        productIndex            n KE = 9.6465e-11
        productIndex          O16 KE = 5.93496e-12
    energy = 2.048e-10
        productIndex            n KE = 1.81936e-10
        productIndex          O16 KE = 2.28635e-11
    energy = 4.096e-10
        productIndex            n KE = 3.39414e-10
        productIndex          O16 KE = 7.0186e-11
    energy = 8.192e-10
        productIndex            n KE = 6.74242e-10
        productIndex          O16 KE = 1.44958e-10
    energy = 1.6384e-09
        productIndex            n KE = 1.57983e-09
        productIndex          O16 KE = 5.85674e-11
    energy = 3.2768e-09
        productIndex            n KE = 2.5643e-09
        productIndex          O16 KE = 7.12501e-10
    energy = 6.5536e-09
        productIndex            n KE = 6.10668e-09
        productIndex          O16 KE = 4.46917e-10
    energy = 1.31072e-08
        productIndex            n KE = 1.19317e-08
        productIndex          O16 KE = 1.1755e-09
    energy = 2.62144e-08
        productIndex            n KE = 2.23661e-08
        productIndex          O16 KE = 3.84825e-09
    energy = 5.24288e-08
        productIndex            n KE = 4.36751e-08
        productIndex          O16 KE = 8.75373e-09
    energy = 1.04858e-07
        productIndex            n KE = 8.84148e-08
        productIndex          O16 KE = 1.64428e-08
    energy = 2.09715e-07
        productIndex            n KE = 1.65083e-07
        productIndex          O16 KE = 4.46324e-08
    energy = 4.1943e-07
        productIndex            n KE = 3.69417e-07
        productIndex          O16 KE = 5.00136e-08
    energy = 8.38861e-07
        productIndex            n KE = 7.71626e-07
        productIndex          O16 KE = 6.72349e-08
    energy = 1.67772e-06
        productIndex            n KE = 1.57664e-06
        productIndex          O16 KE = 1.01086e-07
    energy = 3.35544e-06
        productIndex            n KE = 3.04627e-06
        productIndex          O16 KE = 3.09174e-07
    energy = 6.71089e-06
        productIndex            n KE = 5.68262e-06
        productIndex          O16 KE = 1.02827e-06
    energy = 1.34218e-05
        productIndex            n KE = 1.04481e-05
        productIndex          O16 KE = 2.97366e-06
    energy = 2.68435e-05
        productIndex            n KE = 2.1822e-05
        productIndex          O16 KE = 5.02151e-06
    energy = 5.36871e-05
        productIndex            n KE = 4.54374e-05
        productIndex          O16 KE = 8.24968e-06
    energy = 0.000107374
        productIndex            n KE = 0.000101546
        productIndex          O16 KE = 5.82769e-06
    energy = 0.000214748
        productIndex            n KE = 0.000208588
        productIndex          O16 KE = 6.16005e-06
    energy = 0.000429497
        productIndex            n KE = 0.000359394
        productIndex          O16 KE = 7.01023e-05
    energy = 0.000858993
        productIndex            n KE = 0.000750863
        productIndex          O16 KE = 0.000108131
    energy = 0.00171799
        productIndex            n KE = 0.00165948
        productIndex          O16 KE = 5.85044e-05
    energy = 0.00343597
        productIndex            n KE = 0.00272867
        productIndex          O16 KE = 0.000707304
    energy = 0.00687195
        productIndex            n KE = 0.00649224
        productIndex          O16 KE = 0.000379706
    energy = 0.0137439
        productIndex            n KE = 0.01149
        productIndex          O16 KE = 0.00225388
    energy = 0.0274878
        productIndex            n KE = 0.0247236
        productIndex          O16 KE = 0.00276421
    energy = 0.0549756
        productIndex            n KE = 0.0534769
        productIndex          O16 KE = 0.00149873
    energy = 0.109951
        productIndex            n KE = 0.0955065
        productIndex          O16 KE = 0.0144447
    energy = 0.219902
        productIndex            n KE = 0.190825
        productIndex          O16 KE = 0.0290777
    energy = 0.439805
        productIndex            n KE = 0.374064
        productIndex          O16 KE = 0.0657402
    energy = 0.879609
        productIndex            n KE = 0.74166
        productIndex          O16 KE = 0.137949
    energy = 1.75922
        productIndex            n KE = 1.46346
        productIndex          O16 KE = 0.295757
    energy = 3.51844
        productIndex            n KE = 3.31734
        productIndex          O16 KE = 0.201097
    energy = 7.03687
        productIndex            n KE = 6.43969
        productIndex          O16 KE = 0.597186
    energy = 14.0737
        productIndex            n KE = 11.4091
        productIndex          O16 KE = 2.66465
    energy = 28.1475
        productIndex            n KE = 27.8634
        productIndex          O16 KE = 0.284077
    energy = 56.295
        productIndex            n KE = 56.0668
        productIndex          O16 KE = 0.228156
reaction (  1) = n + (O16_e1 -> O16)  threshold = 6.43088
    energy = 6.43088
        productIndex            n KE = 0.0226898
    energy = 12.8618
        productIndex            n KE = 6.49729
        productIndex       photon KE = 6.12989
    energy = 25.7235
        productIndex            n KE = 19.114
        productIndex       photon KE = 6.12989
    energy = 51.4471
        productIndex            n KE = 38.6032
reaction (  2) = n + (O16_e2 -> O16)  threshold = 6.51646
    energy = 6.51646
        productIndex            n KE = 0.0229925
        productIndex       photon KE = 0.51
        productIndex       photon KE = 0.51
    energy = 13.0329
        productIndex            n KE = 5.46667
        productIndex       photon KE = 6.12989
    energy = 26.0658
        productIndex            n KE = 19.8427
    energy = 52.1317
        productIndex            n KE = 43.9494
reaction (  3) = n + (O16_e3 -> O16)  threshold = 7.3533
    energy = 7.3533
        productIndex            n KE = 0.0259541
    energy = 14.7066
        productIndex            n KE = 7.39105
        productIndex       photon KE = 6.12989
    energy = 29.4132
        productIndex            n KE = 20.9032
    energy = 58.8264
        productIndex            n KE = 48.4155
reaction (  4) = n + (O16_e4 -> O16)  threshold = 7.5656
    energy = 7.5656
        productIndex            n KE = 0.0267058
        productIndex       photon KE = 6.12989
    energy = 15.1312
        productIndex            n KE = 7.52987
    energy = 30.2624
        productIndex            n KE = 22.9425
    energy = 60.5248
        productIndex            n KE = 49.4094
reaction (  5) = n + (O16_e5 -> O16)  threshold = 9.43138
    energy = 9.43138
        productIndex            n KE = 0.0333172
        productIndex       photon KE = 6.12989
    energy = 18.8628
        productIndex            n KE = 9.54408
        productIndex       photon KE = 6.12989
        productIndex       photon KE = 2.74201
    energy = 37.7255
        productIndex            n KE = 22.4819
    energy = 75.451
        productIndex            n KE = 66.1734
reaction (  6) = n + (O16_e6 -> O16)  threshold = 11.648
    energy = 11.648
        productIndex            n KE = 0.0411848
    energy = 23.2959
        productIndex            n KE = 11.8079
    energy = 46.5919
        productIndex            n KE = 26.9007
    energy = 93.1838
        productIndex            n KE = 77.5592
reaction (  7) = n + (O16_e7 -> O16)  threshold = 11.7787
    energy = 11.7787
        productIndex            n KE = 0.0416493
        productIndex       photon KE = 6.12989
    energy = 23.5574
        productIndex            n KE = 10.8399
    energy = 47.1149
        productIndex            n KE = 29.4102
    energy = 94.2298
        productIndex            n KE = 73.1444
reaction (  8) = n + (O16 -> O16 + photon) [continuum]  threshold = 10.1895
    energy = 10.1895
        productIndex            n KE = 0.04569
        productIndex          O16 KE = 0.587031
        productIndex       photon KE = 0.0861765
    energy = 20.3789
        productIndex            n KE = 6.01414
        productIndex          O16 KE = 0.351287
        productIndex       photon KE = 4.62424
        productIndex       photon KE = 4.04568
        productIndex       photon KE = 6.9171
        productIndex       photon KE = 4.1629
    energy = 40.7578
        productIndex            n KE = 15.538
        productIndex          O16 KE = 0.520517
        productIndex       photon KE = 9.87726
    energy = 81.5156
        productIndex            n KE = 19.558
        productIndex          O16 KE = 1.61787
        productIndex       photon KE = 6.17086
        productIndex       photon KE = 5.13878
reaction (  9) = 2n + O15 + photon  threshold = 16.6516
    energy = 16.6516
        productIndex            n KE = 0.0379138
        productIndex            n KE = 0.0666078
        productIndex          O15 KE = 0.807871
    energy = 33.3032
        productIndex            n KE = 6.08978
        productIndex            n KE = 0.45494
        productIndex          O15 KE = 3.39606
    energy = 66.6064
        productIndex            n KE = 1.6022
        productIndex            n KE = 1.3033
        productIndex          O15 KE = 3.6539
reaction ( 10) = n + H1 + N15 + photon  threshold = 12.8924
    energy = 12.8924
        productIndex            n KE = 0.0443666
        productIndex           H1 KE = 0.0724188
        productIndex          N15 KE = 0.685787
    energy = 25.7847
        productIndex            n KE = 3.58067
        productIndex           H1 KE = 3.22057
        productIndex          N15 KE = 0.947132
        productIndex       photon KE = 5.16477
    energy = 51.5694
        productIndex            n KE = 8.42963
        productIndex           H1 KE = 4.52404
        productIndex          N15 KE = 2.15984
        productIndex       photon KE = 5.23543
reaction ( 11) = H1 + N16  threshold = 10.2445
    energy = 10.2445
        productIndex           H1 KE = 0.0361714
        productIndex          N16 KE = 0.574467
    energy = 20.489
        productIndex           H1 KE = 8.01683
        productIndex          N16 KE = 2.83837
        productIndex       photon KE = 0.297
    energy = 40.978
        productIndex           H1 KE = 30.5597
        productIndex          N16 KE = 0.784602
    energy = 81.9561
        productIndex           H1 KE = 66.7199
        productIndex          N16 KE = 5.60269
reaction ( 12) = H1 + (N16_e1 -> N16)  threshold = 10.3722
    energy = 10.3722
        productIndex           H1 KE = 0.0366242
    energy = 20.7444
        productIndex           H1 KE = 8.66098
        productIndex       photon KE = 0.297
        productIndex       photon KE = 0.1201
    energy = 41.4888
        productIndex           H1 KE = 31.4736
    energy = 82.9775
        productIndex           H1 KE = 56.1856
reaction ( 13) = H1 + (N16_e2 -> N16)  threshold = 10.5602
    energy = 10.5602
        productIndex           H1 KE = 0.037291
        productIndex       photon KE = 0.1201
    energy = 21.1205
        productIndex           H1 KE = 7.91119
    energy = 42.241
        productIndex           H1 KE = 27.6749
    energy = 84.4819
        productIndex           H1 KE = 68.9306
reaction ( 14) = H1 + (N16_e3 -> N16)  threshold = 10.6671
    energy = 10.6671
        productIndex           H1 KE = 0.03767
        productIndex       photon KE = 0.297
    energy = 21.3342
        productIndex           H1 KE = 10.5214
    energy = 42.6683
        productIndex           H1 KE = 29.3557
    energy = 85.3366
        productIndex           H1 KE = 70.6474
reaction ( 15) = H2 + N15  threshold = 10.5275
    energy = 10.5275
        productIndex           H2 KE = 0.0742931
        productIndex          N15 KE = 0.553296
    energy = 21.055
        productIndex           H2 KE = 11.13
        productIndex          N15 KE = 0.025103
    energy = 42.11
        productIndex           H2 KE = 21.9672
        productIndex          N15 KE = 10.2431
    energy = 84.22
        productIndex           H2 KE = 53.2556
        productIndex          N15 KE = 21.0649
reaction ( 16) = H2 + (N15_e1 -> N15)  threshold = 16.13
    energy = 16.13
        productIndex           H2 KE = 0.114091
    energy = 32.26
        productIndex           H2 KE = 15.6514
    energy = 64.52
        productIndex           H2 KE = 43.347
reaction ( 17) = H2 + (N15_e2 -> N15)  threshold = 16.161
    energy = 16.161
        productIndex           H2 KE = 0.114311
    energy = 32.322
        productIndex           H2 KE = 14.6578
    energy = 64.644
        productIndex           H2 KE = 44.4778
reaction ( 18) = H2 + (N15_e3 -> N15)  threshold = 17.2502
    energy = 17.2502
        productIndex           H2 KE = 0.12207
    energy = 34.5004
        productIndex           H2 KE = 13.9471
    energy = 69.0008
        productIndex           H2 KE = 35.947
reaction ( 19) = H2 + (N15_e4 -> N15)  threshold = 18.1338
    energy = 18.1338
        productIndex           H2 KE = 0.128369
    energy = 36.2676
        productIndex           H2 KE = 14.942
    energy = 72.5352
        productIndex           H2 KE = 44.3798
reaction ( 20) = H2 + (N15_e5 -> N15)  threshold = 18.289
    energy = 18.289
        productIndex           H2 KE = 0.129476
        productIndex       photon KE = 5.2988
    energy = 36.578
        productIndex           H2 KE = 18.0953
    energy = 73.1561
        productIndex           H2 KE = 38.5039
reaction ( 21) = H2 + (N15_e6 -> N15)  threshold = 18.5718
    energy = 18.5718
        productIndex           H2 KE = 0.131493
    energy = 37.1436
        productIndex           H2 KE = 15.7184
    energy = 74.2872
        productIndex           H2 KE = 49.864
reaction ( 22) = H2 + (N15_e7 -> N15)  threshold = 19.3645
    energy = 19.3645
        productIndex           H2 KE = 0.137149
        productIndex       photon KE = 5.2988
    energy = 38.729
        productIndex           H2 KE = 17.2763
    energy = 77.4581
        productIndex           H2 KE = 49.4433
reaction ( 23) = H2 + (N15_e8 -> N15)  threshold = 19.6394
    energy = 19.6394
        productIndex           H2 KE = 0.139112
        productIndex       photon KE = 5.2988
    energy = 39.2789
        productIndex           H2 KE = 15.0157
    energy = 78.5577
        productIndex           H2 KE = 49.7577
reaction ( 24) = H3 + N14  threshold = 15.3921
    energy = 15.3921
        productIndex           H3 KE = 0.162982
        productIndex          N14 KE = 0.756701
    energy = 30.7841
        productIndex           H3 KE = 8.30823
        productIndex          N14 KE = 8.0035
    energy = 61.5683
        productIndex           H3 KE = 35.1857
        productIndex          N14 KE = 11.9101
reaction ( 25) = H3 + (N14_e1 -> N14)  threshold = 17.8507
    energy = 17.8507
        productIndex           H3 KE = 0.189205
    energy = 35.7014
        productIndex           H3 KE = 13.131
    energy = 71.4029
        productIndex           H3 KE = 43.2107
reaction ( 26) = H3 + (N14_e2 -> N14)  threshold = 19.5891
    energy = 19.5891
        productIndex           H3 KE = 0.207778
        productIndex       photon KE = 2.3128
    energy = 39.1783
        productIndex           H3 KE = 14.4977
    energy = 78.3566
        productIndex           H3 KE = 50.1009
reaction ( 27) = He4 + C13  threshold = 2.35532
    energy = 2.35532
        productIndex          He4 KE = 0.0329215
        productIndex          C13 KE = 0.10695
    energy = 4.71064
        productIndex          He4 KE = 1.54709
        productIndex          C13 KE = 0.948143
    energy = 9.42128
        productIndex          He4 KE = 4.89979
        productIndex          C13 KE = 2.30617
    energy = 18.8426
        productIndex          He4 KE = 12.3052
        productIndex          C13 KE = 4.32224
    energy = 37.6851
        productIndex          He4 KE = 19.2841
        productIndex          C13 KE = 16.1862
    energy = 75.3702
        productIndex          He4 KE = 45.5923
        productIndex          C13 KE = 27.5638
reaction ( 28) = He4 + (C13_e1 -> C13)  threshold = 5.63954
    energy = 5.63954
        productIndex          He4 KE = 0.078933
    energy = 11.2791
        productIndex          He4 KE = 3.56115
    energy = 22.5582
        productIndex          He4 KE = 9.00486
        productIndex       photon KE = 3.6845
    energy = 45.1163
        productIndex          He4 KE = 33.0793
    energy = 90.2327
        productIndex          He4 KE = 77.2957
reaction ( 29) = He4 + (C13_e2 -> C13)  threshold = 6.27217
    energy = 6.27217
        productIndex          He4 KE = 0.0878102
    energy = 12.5443
        productIndex          He4 KE = 5.60912
        productIndex       photon KE = 3.6845
    energy = 25.0887
        productIndex          He4 KE = 15.9661
        productIndex       photon KE = 3.6845
        productIndex       photon KE = 3.0894
    energy = 50.1774
        productIndex          He4 KE = 41.3526
reaction ( 30) = He4 + (C13_e3 -> C13)  threshold = 6.45215
    energy = 6.45215
        productIndex          He4 KE = 0.0903365
    energy = 12.9043
        productIndex          He4 KE = 4.56092
        productIndex       photon KE = 0.1693
    energy = 25.8086
        productIndex          He4 KE = 11.0851
    energy = 51.6172
        productIndex          He4 KE = 36.1343
reaction ( 31) = O17 + photon  threshold = 0
    energy = 1e-13
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 2e-13
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 0.8708
    energy = 4e-13
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 8e-13
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 1.6e-12
        productIndex       photon KE = 3.2722
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 3.2e-12
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 6.4e-12
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 1.28e-11
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 2.56e-11
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 5.12e-11
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 1.024e-10
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 2.048e-10
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 4.096e-10
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 8.192e-10
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 1.6384e-09
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 3.2768e-09
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 6.5536e-09
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 1.31072e-08
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 2.62144e-08
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 5.24288e-08
        productIndex       photon KE = 3.2722
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 1.04858e-07
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 2.09715e-07
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 4.1943e-07
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 8.38861e-07
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 1.67772e-06
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 3.35544e-06
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 6.71089e-06
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 1.34218e-05
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 2.68435e-05
        productIndex       photon KE = 3.2722
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 5.36871e-05
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 0.8708
    energy = 0.000107374
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 0.000214748
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 0.000429497
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 0.000858993
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 0.8708
    energy = 0.00171799
        productIndex       photon KE = 3.2722
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 0.00343597
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 0.00687195
        productIndex       photon KE = 0.8708
    energy = 0.0137439
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 0.0274878
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 0.0549756
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 0.109951
        productIndex       photon KE = 3.2722
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 0.8708
    energy = 0.219902
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 0.439805
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 0.879609
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 1.75922
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 3.51844
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 7.03687
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 0.8708
    energy = 14.0737
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 28.1475
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
    energy = 56.295
        productIndex       photon KE = 2.1842
        productIndex       photon KE = 1.088
        productIndex       photon KE = 0.8708
reaction ( 32) = n + He4 + C12 + photon  threshold = 7.61359
    energy = 7.61359
        productIndex            n KE = 0.0415657
        productIndex          He4 KE = 0.540779
        productIndex          C12 KE = 0.310723
    energy = 15.2272
        productIndex            n KE = 2.05164
        productIndex          He4 KE = 3.56936
        productIndex          C12 KE = 1.3487
    energy = 30.4544
        productIndex            n KE = 18.9863
        productIndex          He4 KE = 6.80786
        productIndex          C12 KE = 2.02158
    energy = 60.9087
        productIndex            n KE = 14.2363
        productIndex          He4 KE = 4.99119
        productIndex          C12 KE = 3.59328
reaction ( 33) = n + 4He4 + photon  threshold = 15.3472
    energy = 15.3472
        productIndex            n KE = 0.110395
        productIndex          He4 KE = 0.267267
        productIndex          He4 KE = 0.128308
        productIndex          He4 KE = 0.812244
        productIndex          He4 KE = 0.112963
    energy = 30.6943
        productIndex            n KE = 6.99214
        productIndex          He4 KE = 0.701115
        productIndex          He4 KE = 2.4792
        productIndex          He4 KE = 0.197629
        productIndex          He4 KE = 0.375052
    energy = 61.3887
        productIndex            n KE = 10.4974
        productIndex          He4 KE = 3.51541
        productIndex          He4 KE = 12.5205
        productIndex          He4 KE = 4.65185
        productIndex          He4 KE = 5.45616
reaction ( 34) = 2He4 + Be9 + photon  threshold = 13.6744
    energy = 13.6744
        productIndex          He4 KE = 0.375196
        productIndex          He4 KE = 0.307211
        productIndex          Be9 KE = 0.675569
    energy = 27.3487
        productIndex          He4 KE = 6.94413
        productIndex          He4 KE = 2.03829
        productIndex          Be9 KE = 0.188057
    energy = 54.6974
        productIndex          He4 KE = 4.75345
        productIndex          He4 KE = 4.21339
        productIndex          Be9 KE = 3.04153
reaction ( 35) = sumOfRemainingOutputChannels  threshold = 0
    energy = 1e-13
    energy = 2e-13
    energy = 4e-13
    energy = 8e-13
    energy = 1.6e-12
    energy = 3.2e-12
    energy = 6.4e-12
    energy = 1.28e-11
    energy = 2.56e-11
    energy = 5.12e-11
    energy = 1.024e-10
    energy = 2.048e-10
    energy = 4.096e-10
    energy = 8.192e-10
    energy = 1.6384e-09
    energy = 3.2768e-09
    energy = 6.5536e-09
    energy = 1.31072e-08
    energy = 2.62144e-08
    energy = 5.24288e-08
    energy = 1.04858e-07
    energy = 2.09715e-07
    energy = 4.1943e-07
    energy = 8.38861e-07
    energy = 1.67772e-06
    energy = 3.35544e-06
    energy = 6.71089e-06
    energy = 1.34218e-05
    energy = 2.68435e-05
    energy = 5.36871e-05
    energy = 0.000107374
    energy = 0.000214748
    energy = 0.000429497
    energy = 0.000858993
    energy = 0.00171799
    energy = 0.00343597
    energy = 0.00687195
    energy = 0.0137439
    energy = 0.0274878
    energy = 0.0549756
    energy = 0.109951
    energy = 0.219902
    energy = 0.439805
    energy = 0.879609
    energy = 1.75922
    energy = 3.51844
    energy = 7.03687
    energy = 14.0737
    energy = 28.1475
    energy = 56.295
        productIndex            n KE = 13.4466
        productIndex       photon KE = 5.34106
